### PR TITLE
Feature/206 applicant career progress

### PR DESCRIPTION
**Pull request by BlancoSebastianEzequiel**

### CODE REVIEW

File: test/graphql/Applicant/mutations/updateCurrentApplicant.test.ts [hunk 1]
```diff
@@ -139,6 +139,22 @@ describe("updateCurrentApplicant", () => {
   });
 
   describe("Errors", () => {
+    it("returns an error if approvedYearCount is a number for a graduated", async () => {
```
Comment (llmanuel) [inline test/graphql/Applicant/mutations/updateCurrentApplicant.test.ts:142]: returns an error if isGraduate and approvedYearCount are defined at the same time

File: src/models/Applicant/ApplicantCareer/Model.ts [hunk 2]
```diff
@@ -1,42 +1,58 @@
...
 
   @Column({
-    allowNull: false,
-    type: INTEGER
+    allowNull: true,
+    type: INTEGER,
+    ...optional(validateIntegerInRange({ min: { value: 0, include: false } }))
   })
-  public creditsCount: number;
+  public approvedYearCount: number;
```
Comment (dylanalvarez) [inline src/models/Applicant/ApplicantCareer/Model.ts:48]: mirando la validacion, esto no es approvedYearCount, es currentCareerYear

File: src/models/Applicant/Interface.ts [hunk 3]
```diff
@@ -2,8 +2,9 @@ import { TLink } from "./Link/Interface";
 import { ICreateFiubaUser, IUser, IUserEditable } from "../User";
 
 export interface IApplicantCareer {
-  code: string;
-  creditsCount: number;
+  careerCode: string;
+  approvedYearCount?: number;
+  approvedSubjectCount?: number;
   isGraduate: boolean;
```
Comment (dylanalvarez) [inline src/models/Applicant/Interface.ts:8]: esta interfaz podria ser algo como

  careerCode: string;
  isGraduate: true;

(or)

  careerCode: string;
  approvedYearCount: number;
  approvedSubjectCount: number;
  isGraduate: false;

Comment (dylanalvarez) [reply src/models/Applicant/Interface.ts:8]: (lo mismo aplica al front, imagino)

**Pull request merged**